README.md

```diff
@@ -102,11 +100,13 @@ This repo is available under the [MIT expat license](https://github.com/PostHog/
 
 Need *absolutely 💯% FOSS*? Check out our [posthog-foss](https://github.com/PostHog/posthog-foss) repository, which is purged of all proprietary code and features.
 
-To learn more, [book a demo](https://posthog.com/talk-to-a-human) or see our [pricing page](https://posthog.com/pricing).
+The pricing for our paid plan is completely transparent and available on [our pricing page](https://posthog.com/pricing). 
+
+## We’re hiring!
 
-### We’re hiring!
+Hey! If you're reading this, you've proven yourself as a dedicated README reader. 
 
-Come help us make PostHog even better. We're growing fast [and would love for you to join us](https://posthog.com/careers).
+You might also make a great addition to our team. We're growing fast [and would love for you to join us](https://posthog.com/careers).
 
 ## Contributors 🦸
```

Comment:
We don't update these automatically anymore, maybe it's time to remove it?

Reply:
I'm [asking](https://posthog.slack.com/archives/C02E3BKC78F/p1744675393557379) and will do another PR if anyone says anything